### PR TITLE
Reference and preview data

### PR DESCRIPTION
**Pull request by r01010010**

### CODE REVIEW

File: packages/caspar-graphics/lib/caspar.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+\import React from 'react'
```
Comment (tobiasandersen) [inline packages/caspar-graphics/lib/caspar.js]: What's this?

File: packages/caspar-graphics/lib/caspar.js [hunk 2]
```diff
@@ -25,7 +25,17 @@ export default class Caspar extends React.Component {
     this.state = {
       state: States.loading,
       data: props.data || getQueryData(),
-      didError: false
+      didError: false,
+      visibleReference: false,
+      referenceImage: null
+    }
+
+    try {
```
Comment (tobiasandersen) [inline packages/caspar-graphics/lib/caspar.js]: Could we do this only in dev mode?

File: packages/caspar-graphics/lib/caspar.js [hunk 3]
```diff
@@ -136,12 +159,19 @@ export default class Caspar extends React.Component {
...
+      didError,
+      visibleReference,
+      referenceImage
+    } = this.state
     const shouldRender =
       !didError && state !== States.willStop && state !== States.stopped
 
     return (
       <div
+        id="transitionGroupContainer"
```
Comment (tobiasandersen) [inline packages/caspar-graphics/lib/caspar.js]: Is this used for anything?

File: packages/caspar-graphics/lib/caspar.js [hunk 4]
```diff
@@ -159,6 +189,19 @@ export default class Caspar extends React.Component {
         <TransitionGroup component={FirstChild}>
           {shouldRender && <Template data={data} state={state} />}
         </TransitionGroup>
+
+        {referenceImage && (
+          <div
+            id="reference"
```
Comment (tobiasandersen) [inline packages/caspar-graphics/lib/caspar.js]: Is this used?

Comment (r01010010) [reply packages/caspar-graphics/lib/caspar.js]: I use the id's as `comments`, just to orientate the developer, but I can take them out.

File: packages/caspar-graphics/lib/caspar.js (hunk 2 above)
Comment (r01010010) [reply packages/caspar-graphics/lib/caspar.js]: the whole toggle thing?

Comment (tobiasandersen) [reply packages/caspar-graphics/lib/caspar.js]: yes, don't want to run the risk of showing it in production.

File: packages/caspar-graphics/lib/caspar.js (hunk 4 above)
Comment (tobiasandersen) [reply packages/caspar-graphics/lib/caspar.js]: You can leave it here, but I think React Dev Tools should cover most needs for it.

**Pull request merged**